endoreg_db/utils/defaults/set_default_center.py

```
@@ -0,0 +1,33 @@
+import subprocess
+from endoreg_db.models.administration.center import center
+from endoreg_db.utils.dataloader
```

Pylint (4.0.2)

[error] Parsing failed: 'invalid syntax (set_default_center, line 3)'

(E0001)